### DISCUSSION TITLE
HTML To RST with Simple Table?

### DISCUSSION THREAD

**Discussion by aarish-lakhani** [jgm/pandoc#9942] [Q&A] [answered]
Hi all,
I'm hoping to convert HTML files to RST files using Pandoc, and I was wondering if there is any way to convert tables to simple table format rather than grid table format. Otherwise, if there is open-source code available to convert from grid to simple table format, that would be awesome (working on this right now in the background)!
Thank you!

**Comment by aarish-lakhani** [comment, marked as answer]
https://github.com/aarish-lakhani/rstGridTabletoSimpleTable

I was able to convert it using some (very bad, very very messy) code! If anyone was hoping to do the same, feel free to check out my approach.

**Comment by jgm** [comment]
The current version of pandoc will use a simple table when the content has the right "shape."  If there is anything fancy (e.g., multiple paragraphs in a cell, column widths, row or colspans), then a gride table will be used.
